File: app/src/main/java/com/shentu/wallpager/mvp/contract/SplashContract.java
package com.shentu.wallpager.mvp.contract;

import com.jess.arms.mvp.IModel;
import com.jess.arms.mvp.IView;
import com.shentu.wallpager.mvp.model.entity.SplashAd;

import io.reactivex.Observable;

public interface SplashContract {
    interface View extends IView {
        void showSplash(SplashAd splashAd);
    }

    interface Model extends IModel {
        Observable getSplashAd();
    }
}

File: app/src/main/java/com/shentu/paper/mvp/contract/SplashContract.kt
package com.shentu.paper.mvp.contract

import com.micro.mvp.IModel
import com.micro.mvp.IView

interface SplashContract {
    interface View : IView {
        fun toMainPage()
        fun startCountDown(total: Int)
        fun showCountDownText(time: Int)
    }

    interface Model : IModel {

    }
}